gutenberg.php

```diff
@@ -156,6 +156,7 @@ function gutenberg_pre_init() {
 	$version = str_replace( '-src', '', $wp_version );
 
 	if ( version_compare( $version, '4.9.8', '<' ) ) {
+		unset( $_GET['activate'] );
```

Comment:
I don't understand, why would you unset this global? I think this could at least do with a comment, this doesn't make much sense to me as someone unfamiliar with how this code works.

Reply:
you're right a screenshot is better than words, so before the unset() :
![no](https://user-images.githubusercontent.com/6878027/46038612-dd83bf00-c10b-11e8-8a39-033069650cb2.png)

after : 

![yes](https://user-images.githubusercontent.com/6878027/46038660-fee4ab00-c10b-11e8-8c05-69f799f6311f.png)

@tofumatt I know it's not a big big pull request but to me this is a bug, you cannot display a confirmation message if it's not the case.

Reply:
@tofumatt I've added comment and put all the thing in 1 commit